tests/__init__.py

```diff
@@ -0,0 +1,5 @@
+from os import path
+from pathlib import Path
+
+_FOLDER = Path(path.abspath(__file__)).parent
```

Comment:
Just a nit: I think we can get save the `os` import here.

```suggestion
_FOLDER = Path(__file__).absolute().parent
```